content/concepts/tokens/nft-reserve-requirements.md

```diff
@@ -0,0 +1,55 @@
+---
+html: nft-reserve-requirements.html
+parent: non-fungible-tokens.html
+blurb: Understand reserve requirements for minting and holding NFTs.
+labels:
+ - Non-fungible Tokens, NFTs
+---
+
+# NFT Reserve Requirements
+
+Minting, holding, and offering NFTs for sale require XRP held in reserve. The reserve charges can add up quickly. Understanding the reserve requirements can help you choose the best approach for your business case.
+
+## Base Reserve
+
+You account must set aside a base reserve, currently 10 XRP. The base reserve XRP amount is subject to change. See [Base Reserve and Owner Reserve](reserves.html#base-reserve-and-owner-reserve).
```

Comment:
is it mean to say "Your account" instead of "You account"?

Reply:
Thanks for catching that.